polly/test/DependenceInfo/reduction_indirect_access.ll

```diff
@@ -0,0 +1,39 @@
+; RUN: opt %loadPolly -basic-aa -polly-print-dependences -polly-allow-nonaffine -disable-output < %s | FileCheck %s
```

Comment:
`-passes=print<polly-dependences>` (see #92918).  I think legacy-pm -polly-print-scops wasn't ported yet, so it should be okay to continue using it for now.

Otherwise, patch looks fine.

Reply:
Please don't forget to update this.

Reply:
Thanks for pointing out Eli. Updated.